### PR TITLE
fix match expiration notification texts

### PR DESCRIPTION
**Pull request by akhrail1996**

### CODE REVIEW

File: lib/t/push_notifications/apns.ex [hunk 1]
```diff
@@ -87,11 +87,18 @@ defmodule T.PushNotifications.APNS do
   end
 
   def build_alert_payload("match_about_to_expire" = type, data) do
-    %{"name" => name} = data
+    %{"name" => name, "gender" => gender} = data
```
Comment (ruslandoga) [inline lib/t/push_notifications/apns.ex:90]: `"gender"` wouldn't exist for jobs scheduled before this version.

File: lib/t/push_notifications/apns.ex [hunk 2]
```diff
@@ -87,11 +87,18 @@ defmodule T.PushNotifications.APNS do
...
+
+    body =
+      case gender do
+        "F" -> dgettext("apns", "Contact her ✨")
+        "M" -> dgettext("apns", "Contact him ✨")
+        "N" -> dgettext("apns", "Contact them ✨")
```
Comment (ruslandoga) [inline lib/t/push_notifications/apns.ex:96]: I prefer templating pronouns / verbs / etc.

```elixir
body = dgettext("Contact %{pronoun} ✨", pronoun: pronoun(gender))
```
```gettext
defp pronoun("F"), do: dgettext("apns", "her")
defp pronoun("M"), do: dgettext("apns", "him")
defp pronoun("N"), do: dgettext("apns", "them")
```

This approach can be used for all messages.

Comment (akhrail1996) [reply lib/t/push_notifications/apns.ex:96]: gettext doesn't seem to like this approach (I had similar problem when fixing "news" localization):

image

Comment (ruslandoga) [reply lib/t/push_notifications/apns.ex]: I missed the domain in my example.

```elixir
body = dgettext("apns", "Contact %{pronoun} ✨", pronoun: pronoun(gender))
```

**Review by ruslandoga — APPROVED** [review]

**Pull request merged**